FBSnapshotTestCase/FBSnapshotTestController.m

```diff
@@ -103,7 +103,11 @@ - (BOOL)compareSnapshotOfViewOrLayer:(id)viewOrLayer
                                error:(NSError **)errorPtr
 {
     if (self.recordMode) {
-        return [self _recordSnapshotOfViewOrLayer:viewOrLayer selector:selector identifier:identifier error:errorPtr];
+        if ([self _saveImagesInRecordMode]) {
```

Comment:
We should also provide a way to set this via a property, via `FBSnapshotTestCase`.

Reply:
then we can remove this one? just set it from there?